Packages/com.unity.inputsystem/InputSystem/Controls/DiscreteButtonControl.cs

```diff
@@ -81,7 +81,7 @@ public override unsafe float ReadUnprocessedValueFromState(void* statePtr)
 
         public override unsafe void WriteValueIntoState(float value, void* statePtr)
         {
-            throw new NotImplementedException();
+            throw new NotSupportedException();
```

Comment:
Why is this not supported? Can we put a reason into the exception?

Reply:
>Why is this not supported?
It's technically not impossible but a bit unclear how exactly this would operate. The control maps multiple input states to a single input value so when you write an input value back, it's unclear what state that should correspond to.

>Can we put a reason into the exception?
Yup, I'll go add a blurb.